File: firstProject/src/main/java/com/project/firstProject/faq/service/FaqService.java
package com.project.firstProject.faq.service;

import java.util.List;

import com.project.firstProject.faq.vo.FaqVO;

public interface FaqService {

	public List<FaqVO> getFaqList(FaqVO faqVo);
	
	public int getFaqListCount(FaqVO faqVo);
	
}

File: firstProject/src/main/resources/static/app/pages/main/faq/faq.service.js
(function(){
	'use strict';
	
	angular.module('myApp.main.faq')
		.service('faqService', faqService);
	
	function faqService($http, config){
		console.info("faqService");
		
		this.getFaqList = function getFaqList(param){
			return $http.get(config.apiUrl + 'faq/getFaqList', {params: param}).then(function(response){
				return response;
			});
		};
		
	};
	
})();